commands/pull_request.go

```diff
@@ -267,12 +266,13 @@ of text is the title and the rest is the description.`, fullBase, fullHead))
 			re := regexp.MustCompile(`\nSigned-off-by:\s.*$`)
 			message = re.ReplaceAllString(message, "")
 		} else if len(commits) > 1 {
+			commitLogs := ""
```

Comment:
Instead of adding this line, can you initialize the `commitLogs` variable by turning the `=` on the next line into `:=` instead?

Reply:
Thanks, have updated this.